exporter/collector/README.md

```diff
@@ -317,6 +318,13 @@ additional flexibility in parsing log severity from incoming entries.
 
 By default, the exporter sends telemetry to the project specified by `project` in the configuration. This can be overridden on a per-metrics basis using the `gcp.project.id` resource attribute. For example, if a metric has a label `project`, you could use the `groupbyattrs` processor to promote it to a resource label, and the `resource` processor to rename the attribute from `project` to `gcp.project.id`.
 
+### Multi-Project quota usage
+
+The `gcp.project.id` label can be combined with the `destination_project_quota` option to attribute quota usage to the project parsed by the label. This feature is currently only available
```

Comment:
If I don't set gcp.project.id, but set `project` to a different value than my credentials' project, will this option still use quota from my destination project, or will it use quota from the credentials' project?

Reply:
Yes, good call out. Updated the description of the collector setting above